### ISSUE TITLE
Matching Game

### ISSUE THREAD

**Issue by Delikatessen** [CSAllenISD/2023-ISP-unIQue#43]
The first and most simple game, you can use https://liveweave.com/ to test JavaScript w/ HTML and CSS live.

This will need:
Visual Accessibility. (For colorblindness, each card's graphic cannot rely on color, use unique shapes)
Sound Accessibility (For people who are visually impaired, give each card a unique sound according to it's graphic)
Difficulty Settings (For replayability)
Meditative Environment (Soft, calming background music and SFX that can be easily turned off)

I'd love to see a working version by the end of this week.

**Comment by MasonG31** [comment]
This sounds great for several reasons:
1) The accessibility for disabilities so that everyone can use our games is probably unlikely, but it is such a great addition. Making all of our games inclusive is morally right and will give our product a nicer feel. It's a win-win.

2) The meditative environment is a double-edged sword that will not only make our game better but will also help with the calming and mental health side of the project, while also fitting the IQ part. This is an excellent addition to the matching game that helps make our project even more valuable to the users.

3) The difficulty settings will make this more captivating and users will be able to play it for more time, rather than being bored after a few minutes.

Sounds great, love it

**Comment by MasonG31** [comment]
I looked at one of the early rough drafts and it already looks great. The graphics and flow of the game are already impressive, and still need more work, but right now the matching game is a home run. Looking forward to the other games we will make.
